src/app/_hooks/use-fcm.ts

```
@@ -0,0 +1,45 @@
+
+    try {
+      const permission = await Notification.requestPermission();
+      console.log("🔐 권한 상태:", permission);
+      if (permission !== "granted") return;
+
+      const messaging = await getFirebaseMessaging();
+      console.log("🔥 messaging 객체:", messaging);
+      const registration = await navigator.serviceWorker.ready;
+
+      const token = await getToken(messaging!, {
+        vapidKey: process.env.NEXT_PUBLIC_FIREBASE_VAPID_KEY!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)